sdk/apis/kubermatic/v1/configuration.go

```diff
@@ -272,6 +272,8 @@ type KubermaticUserClusterConfiguration struct {
 	Addons KubermaticAddonsConfiguration `json:"addons,omitempty"`
 	// SystemApplications contains configuration for system Applications (such as CNI).
 	SystemApplications SystemApplicationsConfiguration `json:"systemApplications,omitempty"`
+	// DefaultApplications contains configuration for default Applications from the ee catalog.
+	DefaultApplications DefaultApplicationsConfiguration `json:"defaultApplications,omitempty"`
```

Comment:
`DefaultApplications` is not the correct name since it's misleading. When we say "default" applications in KKP terms it means the application that are defaulted(similar to enforced applications). These configurations are rather for `defaultApplicationCatalog` :)